src/meta/Dimensions.ts

```diff
@@ -21,9 +21,16 @@ export interface DimensionResults {
 	offset: TopLeft & Size;
 	size: Size;
 	scroll: TopLeft & Size;
+	client: TopLeft & Size;
```

Comment:
Could this be breaking, if a consumer is importing and creating an object using this interface?

Reply:
I guess I'm not sure how since we are only adding data. As long as they are using the DimensionResults type I think they should be OK?

Reply:
Well we export this interface, so if someone decided to import it and use it to construct their own object... adding this non optional property would cause compilation errors. An edge case for sure (and one maybe we don't need to consider) as consumers should just be getting the object from dimensions

```ts
const myDimensions: DimensionResults = {
    size: { /* stuff */},
    offset: { /* stuff */},
    scroll: { /* stuff */},
    position: { /* stuff */}
}
```

Reply:
Oh! Yeah, I get it.  That case would indeed create an error.

Reply:
I should mention that there is no real rush on this, we can wait until we add more breaking changes to include this!